src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -134,7 +122,7 @@ function formatTestValueForUnifiedBranchingType(block: ISelectOneResponseBlock):
     console.warn('Choices are empty for SelectOneBlock, providing `true` by default')
     return 'true'
   }
-  return `OR(${map(blockChoicesKey, (choice) => `block.value = '${choice}'`).join(',')})`
+  return `OR(${map(blockChoicesKey, (choice) => `block.value = "${choice}"`).join(',')})`
```

Comment:
I think we should keep singe quote here to avoid escaping the character.
Mark said in [vmo-4322 description](https://viamoinc.atlassian.net/browse/VMO-4322)
> (Escaping of double-quotes as needed within JSON. You might be able to use single-quotes instead of double-quotes for surrounding the choice names)

Reply:
Ok ok, my thinking on the spot was that we're already sluggifying choice names. So if the choice is `Cat's Korner`, we end up with `cats_korner` in the expression. However, I'm now realizing that we're not the only flow builder that will ever exit! 😹 

To address what you've mentioned, I'm a bit confused: I thought double quotes would be safer? Eg. the following would _not_ break, but with single quotes it would. Am I hearing the Case* you're trying to make properly?
```
block.value = "Cat's Korner"
```

Reply:
You make a point for this `Cat's Korner` example.
I'm just wondering if why we shouldn't escape the double quote here? (as Mark mentioned in vmo-4322 description)

To be honest, I don't really have an idea for now which one is the correct formatting. But, if you're good with double quote, I'm fine. And maybe we need to use double quote for `choices` here https://github.com/FLOIP/flow-builder/pull/170/files#diff-54364b669a225aca3b26fa98446e1ca87c32c769a4db1e764a06e447eff24f60R93 as well.

Reply:
Ok ok, I've updated the other `choices` to be double-quoted.

Reply:
Great 👍 , thanks for this consistency